### ISSUE TITLE
Agreeing our meetings calendar: 2022

### ISSUE THREAD

**Issue by anisa-hawes** [programminghistorian/jekyll#2494]
Hello all,

I would like to propose that we agree our meetings calendar in advance for 2022. From my point of view, this would help to reduce administration and Doodle Polling. Meanwhile, my hope is that this would also offer you greater clarity when forward planning alongside your other commitments. My hope is to make the scheduling simpler and more transparent. Please remember, there is no expectation that everyone will be able to attend every meeting relevant to them. I will always share minutes internally, so that anyone unable to attend can still understand what was discussed.

We currently have four core meeting sequences:

1. Project Team Meetings
2. Board of Trustees Meetings
3. Managing Editors' Meetings
4. Annual General Meeting

I'd like to propose that we agree to schedule 1., 2., and 3. on a once per quarter basis. In each sequence I will alternate the day of the week, to increase the probability that everyone will be able to attend at least some of the meetings.  

The Managing Editors have previously expressed their wish to meet the week following the Project Team Meetings, but of course I welcome their suggestions for an alternative pattern if the schedule set out below doesn't suit their needs.

Meanwhile, our Annual General Meeting will continue to occur once per year, in May.

I've reviewed the list of holidays/religious days you collectively complied [here](https://github.com/programminghistorian/jekyll/issues/1739). I don't think this calendar presents obvious clashes, but please let me know if you feel one of the dates I've suggested feels unsuitable. 

What are your thoughts on this proposed schedule?

2022
---

**January**
(no meetings)

**February**
Friday 18th: Board of Trustees Meeting [14:00 UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20220218T140000&p1=1440) See also: #2478 

**March**
Friday 11th: Project Team Meeting [13:00 UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20220311T130000&p1=1440) See also: #2486 
Wednesday 16th: Managing Editors' Meeting [14:00 UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20220316T140000&p1=1440&p2=43&p3=232&p4=195&p5=133)

**April**
(no meetings)

**May**
Thursday 19th: Board of Trustees Meeting [14:00 UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20220519T140000&p1=1440)
Monday 23rd: Annual General Meeting [15:00 UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20220523T150000&p1=1440)

**June**
Friday 17th: Managing Editors' Meeting [14:00 UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20220617T140000&p1=1440&p2=43&p3=232&p4=198&p5=195&p6=133)

**July**
Thursday 14th: Board of Trustees Meeting [14:00 UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20220714T140000&p1=1440)

**August**
(no meetings)

**September**
Friday 9th: Project Team Meeting [15:00 UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20220909T150000&p1=1440)
Wednesday 14th: Managing Editors' Meeting [14:00 UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20220914T140000&p1=1440&p2=198&p3=232&p4=195&p5=133)

**October**
(no meetings)

**November**
Thursday 17th: Board of Trustees Meeting [14:00 UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20221117T140000&p1=1440)

**December**
Thursday 15th: Managing Editors' Meeting [14:00 UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20221215T140000&p1=1440&p2=198&p3=232&p4=195&p5=133)

**Comment by drjwbaker** [comment]
Thanks @anisa-hawes. We've resisted before because of not knowing timetables for things like teaching until nearer term, but I much prefer this than doodling each time to find I can't make it anyway. Three thoughts:

- I propose moving the ProgHist BoD from August to July.
- Will you (or someone else) create calendar invites for all these? (and/or put them on the shared gmail calendar?) I just know if I don't have a calendar invite I'll forget to come.
-  We may need to be flexible to add special meetings: for example, I know @ZoeLeBlanc and the tech team have big/exciting infrastructure plans, and they may feel someone - say Zoe - has to be at a meeting at which something time sensitive is discussed, but they can't make the allocated date/time. I suggest in those instances that we do, say, the Project Team meeting as planned, then have a single-issue special meeting for key business that key people can't make. Good idea?

**Comment by anisa-hawes** [comment]
Thank you, @drjwbaker. These are very useful points! 

- I've adjusted the August BoT Meeting date (above), suggesting this on 15th July instead.
- Yes. Maybe someone could work with me/grant me temp access so that we can send initial invites from the shared Gmail? Going forward to when I have my PH email set up, I'll be able to create a shared calendar, send meeting invites, and host the calls (I gather that the Google Workspace package includes Google Meet).
- I think that's a very good idea re: single-issue meetings. I'm very happy to be flexible and add these in as needed. 
- One other thought I had is that I know there is a required minimum number (3) people who need to attend the BoT Meetings. In a case where fewer than 3 can make it on a scheduled date, we will of course need to adjust the date/time of those.

**Comment by drjwbaker** [comment]
Belatedly, many thanks. On the second point, we seem to be getting closer to resolving your email issue, so can be park this and return to it (if not resolved) after the March meetings?

**Comment by anisa-hawes** [comment]
Yes, good idea. I'll come back to this in late March/early April when my PH email is set up. At that point, I'll send advance invites for all our 2022 meetings. 

I've set myself a reminder to return to this Issue during the week beginning 21st March.

**Comment by drjwbaker** [comment]
@anisa-hawes: with apologies if you have this in hand, is a date for the AGM close to being settled?

**Comment by anisa-hawes** [comment]
Thank you, @drjwbaker. I haven't scheduled this yet. Is there a particular date(s) you are unable to join? Let me know.

(I've actually been prepping some calendar invitations this afternoon, but I seem to have hit a Calendar Use Limit when I try to send an event invitation to the Google Group (37 members). I'm not certain, but it could be that our Google Workspace is subject to a trial phase.... I'm trying to work our what is stopping me from being able to do this...)

**Comment by drjwbaker** [comment]
Late-May is (currently!) looking okay.

(ack!)

**Comment by anisa-hawes** [comment]
For the AGM, I'd like to suggest [Monday 23rd](https://www.timeanddate.com/worldclock/converter.html?iso=20220523T150000&p1=1440) or [Tuesday 24th May at 15:00 UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20220524T150000&p1=1440). I have opened another Issue  #2520 for us to agree the date, organise invitations, and plan our agenda.

**Comment by anisa-hawes** [comment]
I have my organisational email address and I have started the process of sending calendar invitations for our 2022 meetings. 

One unexpected sticking point is that the Google Workspace imposes limits on number of invitations I can send to a single event, and the number of invitations I can send in a single day! So a little inconvenient... but I think I can work it out by doing them in batches. Closing this Issue now.
